spec/rubocop/remote_config_spec.rb

```diff
@@ -189,4 +189,17 @@
       end
     end
   end
+
+  describe '.inherit_from_remote' do
+    context 'when the remote includes file starting with `./`' do
+      let(:includes_file) { './base.yml' }
+
+      it 'returns remote includes URI' do
+        remote_config = described_class.new(remote_config_url, base_dir)
+        includes_config = remote_config.inherit_from_remote(+includes_file, remote_config_url)
```

Comment:
I think it cannot be expected to be duplicated always.

```suggestion
        includes_config = remote_config.inherit_from_remote(includes_file, remote_config_url)
```

Reply:
Fixed in 60e6f47 😃